### PR TITLE
Merge changes for the Euro-Ameriga (mostly Amazonie and Beornia) to main 

### PR DESCRIPTION
**Pull request by Kuri21079**
This pull requests includes flavor for Amazonie and Beornia

- Beornia has a journal relating to the differences between King Edward's and the Cattle Barons' ambitions for the country. 
- Beornia has decisions to purchase the Falklands, hold a pan-Anglo summit if England is independent and to share a market with them, unite with Lotharingia, and a journal for claiming all of England.
 - There is also the decision to become the Holy Britannian Empire if fantasy formables rule is enabled. This country has its own flavor too.

- Most of Amazonie's content involves slavery and racial tensions. Amazonie is now part of the Egalitarian Society objective.
- Equinoxea is now a playable puppet of Amazonie that starts out with slavery abolished and universal suffrage.  Amazonie has a decision to annex it once the truce expires.

### CODE REVIEW

File: localization/english/dvg_names_l_english.yml [hunk 1]
```diff
@@ -634,6 +634,10 @@ Lysandre: "Lysandre"
 Lavisse: "Lavisse"
 Gaius: "Gaius"
 Odysseus: "Odysseus"
+Falcone: "Falcone"
```
Comment (MadJamez) [inline localization/english/dvg_names_l_english.yml:637]: Duplicate localization key. Key 'Falcone' is defined in both 'localization/english/dvg_names_l_english.yml' and 'localization/english/dvg_names_l_english.yml'.

**Review by MadJamez — DISMISSED** [review]
[20:55:46][pdx_persistent_reader.cpp:252]: Error: "Unexpected token: state_muskogean_standard_of_living_add, near line: 1376" in file: "common/modifiers/content_1_modifiers.txt" near line: 1376
[20:55:46][pdx_persistent_reader.cpp:252]: Error: "Malformed token: -.05, near line: 8" in file: "common/modifiers/dvg_ama_modifiers.txt" near line: 8
[20:55:46][pdx_persistent_reader.cpp:252]: Error: "Unexpected token: interest_group_ig_petty_bougeoisie_pol_str_mult, near line: 42" in file: "common/modifiers/dvg_beo_modifiers.txt" near line: 42
[20:55:46][culture_type.cpp:126]: Missing localization key for name de_Saint-Andr in culture amazonien
[20:55:46][culture_type.cpp:126]: Missing localization key for name Yeng in culture qingqiuan
[20:55:46][technology_type.cpp:126]: Technology dvgaustralia has no ai_weight set, so AI can't research it
[20:55:47][pdx_persistent_reader.cpp:252]: Error: "Unexpected token: Nova, near line: 3" in file: "common/country_definitions/dvg_ameriga.txt" near line: 3
[20:55:48][pdx_persistent_reader.cpp:252]: Error: "Unexpected token: exists, near line: 1520" in file: "common/dynamic_country_names/dvg_dynamic_country_names.txt" near line: 1520
[20:55:48][pdx_persistent_reader.cpp:252]: Error: "Unexpected token: exists, near line: 1543" in file: "common/dynamic_country_names/dvg_dynamic_country_names.txt" near line: 1543
[20:55:48][gamedatabase.h:314]: Overriding entry 'TWT' for database 'common/dynamic_country_names' in ' file: common/dynamic_country_names/dvg_dynamic_country_names.txt line: 1593'
[20:55:48][gamedatabase.h:314]: Overriding entry 'BEO' for database 'common/dynamic_country_names' in ' file: common/dynamic_country_names/dvg_dynamic_country_names.txt line: 1798'
[20:55:49][pdx_persistent_reader.cpp:252]: Error: "Unexpected token: ai_chance, near line: 91" in file: "events/dvg_england.txt" near line: 93

**Comment by Kuri21079** [comment]
Most of these should be resolved now, though I'm not so sure if the muskogean standard of living and the dvgaustralia tech relates to this branch?

File: common/journal_entries/dvg_ameriga.txt [hunk 2]
```diff
@@ -0,0 +1,99 @@
+
+
+
+je_ameriga_unite = {
```
Comment (MadJamez) [inline common/journal_entries/dvg_ameriga.txt:4]: File 'common/journal_entries/dvg_ameriga.txt' should be in utf8-bom encoding

File: common/journal_entries/dvg_ameriga.txt [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+		trigger_event = { id = dvg_ameriga.3 popup = yes }
+		
+    }
+	fail = {
+	
+	AND = {
+	
+	any_interest_group = { is_insurrectionary = yes }
+	
+	turmoil > 0.9
```
Comment (MadJamez) [inline common/journal_entries/dvg_ameriga.txt:83]: Inconsistent trigger scopes (country vs. state) in file: common/journal_entries/dvg_ameriga.txt line: 83

File: common/journal_entries/dvg_beornia.txt [hunk 4]
```diff
@@ -0,0 +1,829 @@
+﻿
```
Comment (MadJamez) [inline common/journal_entries/dvg_beornia.txt:1]: [10:25:22][gamedatabase.h:314]: Overriding entry 'je_ameriga_conquer' for database 'common/journal_entries' in ' file: common/journal_entries/dvg_beornia.txt line: 490'
[10:25:22][gamedatabase.h:314]: Overriding entry 'je_world_conquer' for database 'common/journal_entries' in ' file: common/journal_entries/dvg_beornia.txt line: 584'

File: common/history/characters/0_dvg_tawantinsuyu.txt [hunk 5]
```diff
@@ -69,5 +72,22 @@
 #				exile_character = yes
 #			}
 		}
+		create_character = {
```
Comment (MadJamez) [inline common/history/characters/0_dvg_tawantinsuyu.txt:75]: [10:25:33][jomini_script_system.cpp:263]: Script system error!
  Error: Failed to scope to culture by key 'Japanese'
  Script location: file: common/history/characters/0_dvg_tawantinsuyu.txt line: 79

[10:25:33][jomini_script_system.cpp:263]: Script system error!
  Error: Event target link 'cu' returned an unset scope
  Script location: file: common/history/characters/0_dvg_tawantinsuyu.txt line: 79

**Review by MadJamez — CHANGES_REQUESTED** [review]
Can you also in general fix formatting? It's al over the place which makes it hard to spot issues an to review code.

File: localization/english/dvg_flavor_beornia_l_english.yml [hunk 6]
```diff
@@ -1,11 +1,454 @@
...
...ia is by all metrics the youngest nation in Ameriga as it was only claimed by English settlers in the 17th century crossing the Odon river, living in small towns often unknown by the Burgundian colonial administration. Over time, they developed a peculiar mindset of exiles distrustful of foreigners and keen to protect their language and culture with the wealthier among them being known as [SCOPE.gsInterestGroup('cattle_barons_ig').GetName]. \n\n One of these [SCOPE.gsInterestGroup('cattle_barons_ig').GetName], Harold Cavendish, claimed descent from the House Cavendish that once held the title of 'Duke of Newcastle'. His house was recognized to rule over these English-settled areas by the Burgundian authorities in 1709 for his efforts in destroying the Anglois settlement of Nouveau Havre that once inhabited the area. Now, with the Burgundians gone thanks to [ROOT.GetCountry.GetRuler.GetFullName] seeing the opportunity in helping Lotharingia's ruling class in their civil war and fight for independence, there are some in the government who seek to use this as a foundation to expand [ROOT.GetCountry.GetName]'s influence further; whether it be annexing new lands or reclaiming old ones."
```
Comment (MadJamez) [inline localization/english/dvg_flavor_beornia_l_english.yml:53]: Maybe make the first Beornia here also [ROOT.GetCountry.GetName] ?

File: localization/english/country_flavor_text_l_english.yml [hunk 7]
```diff
@@ -99,3 +103,10 @@
  DON_FLAVOR_TEXT:0 "The Donetsian host is enjoying freedom, having conquered their own lands from the Crimean Khanate. While border raids still happen it will need a change in the global situation to see the Cossacks hosts on the warpath."
  VOR_FLAVOR_TEXT:0 "The Don Cossacks have recognized the Prince of Ryazan as their liege."
  AKH_FLAVOR_TEXT:0 "The Akhtuba host was founded as the last free Cossack host, on the ruins of the Chayka rebellion and the Turgut attack on the Astrakhani."
+ 
+ AFG_FLAVOR_TEXT:0 "The Karlani Shahdom is this timeline's greater Afghanistan, plus Punjab. It is situated at the crossroads of Persia and India. Who knows what the future holds for this landlocked, mountainous nation?"
+ IRQ_FLAVOR_TEXT:0 "Iraq is a small breakaway state in the middle east, it was able to secure its independence in the collapse of the Ottoman Empire, who now only borders them along a small strip of land in the north, it also borders Al Haasa to the south and the new Shekhbizin Empire to the east. Being sandwiched between the Arab and Persian homelands, how will Iraq define itself?"
```
Comment (MadJamez) [inline localization/english/country_flavor_text_l_english.yml:108]: Seems to have some duplicate flavor text?

File: localization/english/country_flavor_text_l_english.yml [hunk 8]
```diff
@@ -3,8 +3,12 @@
  REV_GENERIC_FLAVOR_TEXT:1 "Dissatisfied with the direction that the so-called organized government was taking, these revolutionaries have decided to take up arms to overthrow it. Will they succeed in seizing power or will they be crushed like so many forgotten revolutions of the past?"
```
Comment (MadJamez) [inline localization/english/country_flavor_text_l_english.yml:3]: Duplicate localizations
[12:25:19][pdx_localize.cpp:279]: Duplicate localization key. Key 'IRQ_FLAVOR_TEXT' is defined in both 'localization/english/country_flavor_text_l_english.yml' and 'localization/english/country_flavor_text_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'DRV_ADJ' is defined in both 'localization/english/dvg_countries_l_english.yml' and 'localization/english/dvg_countries_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'Kiya' is defined in both 'localization/english/dvg_flavor_persia_l_english.yml' and 'localization/english/dvg_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAJA_CALIFORNIA_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAJA_CALIFORNIA_port' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAJA_CALIFORNIA_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAJA_CALIFORNIA_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAJA_CALIFORNIA_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_DURANGO_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_DURANGO_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_DURANGO_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_DURANGO_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_RIO_GRANDE_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_RIO_GRANDE_port' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_RIO_GRANDE_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_RIO_GRANDE_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_RIO_GRANDE_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_ZACATECAS_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_ZACATECAS_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_ZACATECAS_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_ZACATECAS_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAJIO_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAJIO_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAJIO_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAJIO_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_OAXACA_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_OAXACA_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_OAXACA_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_OAXACA_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_OAXACA_port' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_SINALOA_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_SINALOA_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_SINALOA_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_SINALOA_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_SINALOA_port' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_JALISCO_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_JALISCO_port' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_JALISCO_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_JALISCO_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_JALISCO_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_GUERRERO_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_GUERRERO_port' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_GUERRERO_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_GUERRERO_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_GUERRERO_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_MEXICO_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_MEXICO_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_MEXICO_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_MEXICO_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_VERACRUZ_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_VERACRUZ_port' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_VERACRUZ_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_VERACRUZ_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_VERACRUZ_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAHIA_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAHIA_port' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAHIA_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAHIA_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_BAHIA_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_SANTA_CATARINA_city' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_SANTA_CATARINA_port' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_SANTA_CATARINA_mine' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_SANTA_CATARINA_farm' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'HUB_NAME_STATE_SANTA_CATARINA_wood' is defined in both 'localization/english/dvg_hub_names_l_english.yml' and 'localization/english/hub_names_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'STATE_BAJA_CALIFORNIA' is defined in both 'localization/english/dvg_states_l_english.yml' and 'localization/english/map/states_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'STATE_RIO_GRANDE' is defined in both 'localization/english/dvg_states_l_english.yml' and 'localization/english/map/states_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'STATE_GUERRERO' is defined in both 'localization/english/dvg_states_l_english.yml' and 'localization/english/map/states_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'STATE_DURANGO' is defined in both 'localization/english/dvg_states_l_english.yml' and 'localization/english/map/states_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'STATE_JALISCO' is defined in both 'localization/english/dvg_states_l_english.yml' and 'localization/english/map/states_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'STATE_BAHIA' is defined in both 'localization/english/dvg_states_l_english.yml' and 'localization/english/map/states_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'STATE_SANTA_CATARINA' is defined in both 'localization/english/dvg_states_l_english.yml' and 'localization/english/map/states_l_english.yml'.
[12:25:20][pdx_localize.cpp:279]: Duplicate localization key. Key 'STATE_BAJIO' is defined in both 'localization/english/dvg_states_l_english.yml' and 'localization/english/map/states_l_english.yml'

File: common/flag_definitions/dvg_flag_definitions.txt [hunk 9]
```diff
@@ -2559,5 +3233,52 @@ TRC = {
...
+		coa = TRC_republic
+		subject_canton = TRC_republic
+		priority = 10
+		trigger = { 
+			coa_def_republic_flag_trigger
+		}
+	}
 }
 
+CAU = { #Caucasia
```
Comment (MadJamez) [inline common/flag_definitions/dvg_flag_definitions.txt:3246]: "Unexpected token: CAU, near line: 3246" in file: "common/flag_definitions/dvg_flag_definitions.txt" near line: 3284

File: common/history/characters/0_dvg_timurtash.txt [hunk 10]
```diff
@@ -0,0 +1,63 @@
+﻿CHARACTERS = {
```
Comment (MadJamez) [inline common/history/characters/0_dvg_timurtash.txt:1]: [12:26:08][pdx_persistent_reader.cpp:252]: Error: "Wrong type parsed in create_character effect: Artuqid, near line: 24
Wrong type parsed in create_character effect: =, near line: 24" in file: "common/history/characters/0_dvg_timurtash.txt" near line: 35
[12:26:11][jomini_script_system.cpp:263]: Script system error!
  Error: set_owner_of_provinces effect [ Province 'x809040' crossing multiple state regions ]
  Script location: file: events/dvg_persia.txt line: 127

File: common/history/characters/0_dvg_iraq.txt [hunk 11]
```diff
@@ -0,0 +1,21 @@
+﻿CHARACTERS = {
```
Comment (MadJamez) [inline common/history/characters/0_dvg_iraq.txt:1]: [12:26:11][character_templates_utils.cpp:29]: Invalid Ideology ideology_zaydani in  file: common/history/characters/0_dvg_iraq.txt line: 3
[12:26:11][character_templates_utils.cpp:29]: Can only do leader Ideologies in  file: common/history/characters/0_dvg_iraq.txt line: 3
[12:26:11][jomini_effect.cpp:128]: PostValidate of effect 'create_character' returned false at  file: common/history/characters/0_dvg_iraq.txt line: 3
[12:26:11][jomini_script_system.cpp:263]: Script system error!
  Error: add_ideology effect [ Invalid database object 'paternalistic' ]
  Script location: file: common/history/countries/0_dvg_middle_east.txt line: 97

[12:26:11][jomini_effect.cpp:128]: PostValidate of effect 'add_ideology' returned false at  file: common/history/countries/0_dvg_middle_east.txt line: 97
[12:26:11][jomini_script_system.cpp:263]: Script system error!
  Error: has_template trigger [ Invalid database object 'agitator_louise_michel' ]
  Script location: file: common/scripted_effects; create_paris_commune_characters line: 131

**Review by MadJamez — CHANGES_REQUESTED** [review]
This includes more then Beornia? Maybe try to keep it on different branches next time?
Also update the review description so its up to date with the actually content in here.

**Pull request closed without merging**